### ISSUE TITLE
[pi] First iteration of automated accessibility checks

### ISSUE THREAD

**Issue by m8ttyB** [mozilla-lockbox/lockbox-extension#679]
Integrate PI's axe-core api into the integration suite to provide regression checks for accessibility requirements. The PR will provide an opening for discussions around enhancements we may wish to make to the app as will as provide information on which tests should be included in the TravisCI build or offloaded to PI's Jenkins env.

Important as well, this will provide information on what should block the build vs. act as information.

**Comment by m8ttyB** [comment]
This is close to completion, let's keep this as one of our closing closing out tasks for the pivot.

**Comment by devinreams** [comment]
>This is close to completion

@m8ttyB @kimberlythegeek Awesome! I'll add it to "in progress" so it doesn't accidentally get overlooked.

**Comment by devinreams** [comment]
Closed by #709
